crates/typst/src/model/heading.rs

```diff
@@ -46,7 +46,7 @@ use crate::util::{option_eq, NonZeroExt};
 /// can be set to configure the starting depth.
 #[elem(Locatable, Synthesize, Count, Show, ShowSet, LocalName, Refable, Outlinable)]
 pub struct HeadingElem {
-    /// The absolute nesting depth of the heading, starting from one. If set
+    /// The absolute nesting depth of the heading, starting from `{1}`. If set
```

Comment:
why this change?

Reply:
I think this actually reduces legibility. Going from words to numbers makes your brain do a context-switch. This is why it's good style in many languages to spell out small numbers, e.g. up to 12.

Reply:
Really? Well, I guess we hit the subjectivity ceiling.